File: exercicios-java/estrutura-sequencial/exe05.java
import java.util.Scanner;
/*
Faça um Programa que converta metros para centímetros.
*/

public class exe5{

  public static void main(String[] args){
    Scanner leitor = new Scanner(System.in);

    float metros=0, centimetros=0;

    System.out.print("Informe o comprimento em metros: ");
    metros = leitor.nextFloat();

    centimetros = metros * 100;

    System.out.printf("\n%.2f m convertido para centimetros é %.2f cm.\n",metros, centimetros);


  }
}

File: exercicios-javascript/exe05.js

/*
Faça um Programa que converta metros para centímetros.
*/

const readline = require('readline-sync')

let metros = readline.questionFloat('Informe o comprimento em metros: ')

let centimetros = metros * 100

console.log(`${metros}m convertido para centimetros é ${centimetros}cm`)
